addons/survey/views/survey_survey_views.xml

```diff
@@ -172,7 +172,7 @@
                                         </div>
                                     </div>
                                 </group>
-                                <group string="Live Session" name="live_session" invisible="not session_available">
+                                <group string="Live Session" name="live_session" invisible="not session_available or certification">
```

Comment:
Hello @jasa-odoo ,
Would it be possible to update `_compute_session_available` to take `certification` into account?

This way we can clean the other cases in views where we do use `certification` for this visibility check, and be more robust for the future as there would be only a single method to update if we need to change anything?

Reply:
Hello @flch-odoo ,
Thanks for review and I made the suggested changes.
can you please have a look?